src/components/Input.vue

```diff
@@ -0,0 +1,20 @@
+<script setup lang="ts">
+const { modelValue } = defineModel<{
+  modelValue: string
+}>()
+</script>
+
+<template>
+  <input
+    id="input"
+    v-model="modelValue"
+    type="text"
+    v-bind="$attrs"
+    p="x-4 y-2"
+    w="250px"
+    text="center"
+    bg="transparent"
+    border="~ rounded gray-200 dark:gray-700"
+    outline="none active:none"
+  >
+</template>
```

Comment:
is this component necessary? 
- why no just add defineModel on index page? or
- move attrs to index page: you're binding any attr to the component

Reply:
- v-model requires two SFC files, parent and child. So I have to create a child component to demonstrate that.
- It's just an example of extracting input style to an SFC.

Reply:
ok